### PR TITLE
Chapter 7 Outro 1 

### LINKED ISSUE #829: Chapter 7 Outro Page
Implement the outro page for chapter 7

image

https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=7100%3A70217&mode=dev

### PR DESCRIPTION
**Pull request by 0tuedon**
This pr closes #829 
Also having a weird development bug with hydration
Screenshot 2024-02-01 at 18 42 17

### CODE REVIEW

File: content/lessons/chapter-7/outro-1.tsx [hunk 1]
```diff
@@ -17,21 +17,18 @@ export default function Outro1({ lang }) {
 
   return (
     <ChapterEnd
-      className="top-[70px]"
+      className="bottom-[0px]"
```
Comment (benalleng) [inline content/lessons/chapter-7/outro-1.tsx:20]: I think You can just remove this className, I will look to see why it exists in chapter 4

File: content/lessons/chapter-7/outro-1.tsx [hunk 2]
```diff
@@ -4,8 +4,8 @@ import { useTranslations } from 'hooks'
 import { ChapterEnd } from 'ui'
 
 export const metadata = {
-  title: 'chapter_four.outro_one.title',
-  image: '/assets/images/chapter-4-finish.jpg',
+  title: 'chapter_seven.outro_one.title',
+  image: '/assets/images/chapter-7-outro.jpg',
   theme: 'bg-transparent/20',
   secondaryTheme: 'bg-[#472B26]',
   gradientTheme: 'from-[#472B26] via-[rgba(71, 43, 38, 0)]',
```
Comment (benalleng) [inline content/lessons/chapter-7/outro-1.tsx:11]: Lets make the color used be this `#52555d` instead

**Review by benalleng — CHANGES_REQUESTED** [review]
Small change, otherwise looks good!

File: content/lessons/chapter-7/outro-1.tsx (hunk 2 above)
Comment (benalleng) [reply content/lessons/chapter-7/outro-1.tsx:11]: so for the second line that would be `from-[#52555d] via-[#52555dBF]`

Comment (0tuedon) [reply content/lessons/chapter-7/outro-1.tsx:11]: the secondary theme?

**Review by benalleng — APPROVED** [review]

**Comment by benalleng** [comment]
We'll look further into the hydration error after merging

**Pull request merged**